components/automate-ui/src/app/page-components/services-sidebar/services-sidebar.component.ts

```diff
@@ -45,22 +51,23 @@ export class ServicesSidebarComponent implements OnInit {
 
     this.svcHealthSummary$ = this.store.select(createSelector(serviceGroupState,
       (state) => state.servicesHealthSummary));
-    this.svcHealthSummary$.subscribe((servicesHealthSummary) => {
+    this.svcHealthSummary$.pipe(takeUntil(this.isDestroyed)).subscribe((servicesHealthSummary) => {
```

Comment:
This component and sidebar component had a few subscriptions that never got closed when navigating away from *Applications* view.

https://github.com/chef/automate/pull/367/commits/54d183cbc920047c9e23c64f84e153fa5f3f310b
https://github.com/chef/automate/pull/367/commits/ecb68c7da1add60df64768ef9a8b960687449baf